File: FrontEnd/src/app/features/billing/receipts/classes/view-models/list/receipt-list-vm.ts
import { DocumentTypeVM } from './documentType-vm'
import { SimpleEntity } from '../../../../../../shared/classes/simple-entity'

export interface ReceiptListVM {

    invoiceId: string
    date: string
    formattedDate: string
    customer: SimpleEntity
    documentType: DocumentTypeVM
    shipOwner: SimpleEntity
    paymentMethod: SimpleEntity
    invoiceNo: number
    grossAmount: number
    remarks: string
    isEmailSent: boolean

}

File: API/Features/Billing/Receipts/ViewModels/List/ReceiptListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Billing.Receipts {

    public class ReceiptListVM {

        public string InvoiceId { get; set; }
        public string Date { get; set; }
        public int InvoiceNo { get; set; }
        public SimpleEntity Customer { get; set; }
        public SimpleEntity DocumentType { get; set; }
        public SimpleEntity ShipOwner { get; set; }
        public SimpleEntity PaymentMethod { get; set; }
        public decimal GrossAmount { get; set; }
        public string Remarks { get; set; }
        public bool IsEmailSent { get; set; }
        public bool IsCancelled { get; set; }

    }

}